pkg/storaged/stratis-set-key.py

```diff
@@ -0,0 +1,9 @@
+import dbus
```

Comment:
Note to self: The storage page already assumes dbus-python dependency in pkg/storaged/nfs-mounts.py. This script still feels like a hilarious hack -- it ought to be possible to open an fd and pass it to a D-Bus call through the cockpit API somehow.

Reply:
Yeah..., the bridge can receive fds in a reply, but not send them (last time I checked).